src/main/java/fact/coordinates/EarthLocation.java

```diff
@@ -0,0 +1,42 @@
+package fact.coordinates;
+
+/**
+ * Created by maxnoe on 22.05.17.
+ */
+public class EarthLocation {
+    final private double latitudeRad;
+    final private double longitudeRad;
+    final private double altitudeMeter;
+
+    public EarthLocation(double latitudeRad, double longitudeRad, double altitudeMeter) {
+        this.latitudeRad = latitudeRad;
+        this.longitudeRad = longitudeRad;
+        this.altitudeMeter = altitudeMeter;
+    }
+
+    public static EarthLocation fromDegrees(double latitudeDeg, double longitudeDeg, double altitudeMeter) {
```

Comment:
I think it makes more sense to provide another `fromRads` constructor and make the cosntructor private. Seems to be more consistent.